src/Resources/contao/classes/RealEstateImporter.php

```diff
@@ -1150,38 +1152,26 @@ public function getFieldData($field, $group)
                             $results[$i] = serialize($tmp);
                         }
                         break;
-                    case '[1]':
-                        // Returns the first child nodes name.
-                        $index = 0;
-                        foreach ($xmlNode->children() as $c) {
-                            if ($index === 0) {
-                                $results[$i] = $c->getName();
-                            }
-                            $index++;
-                        }
-                        break;
-                    case '[2]':
-                        // Returns the first child nodes name.
-                        $index = 0;
-                        foreach ($xmlNode->children() as $c) {
-                            if ($index === 1) {
-                                $results[$i] = $c->getName();
-                            }
-                            $index++;
-                        }
-                        break;
-                    case '[3]':
-                        // Returns the first child nodes name.
-                        $index = 0;
-                        foreach ($xmlNode->children() as $c) {
-                            if ($index === 2) {
-                                $results[$i] = $c->getName();
+                    case '[':
+                        if(is_numeric($attr_field)) {
+                            // Returns the first child nodes name.
+                            $index = 0;
+                            $attr_field = intval($attr_field) - 1;
+                            foreach ($xmlNode->children() as $c) {
+                                if ($index === $attr_field) {
+                                    $results[$i] = $c->getName();
+                                }
+                                $index++;
                             }
-                            $index++;
+                        } else {
+                            // Returns the value of an multiple XML element.
+                            if(current($attributes)['feldname'] == $attr_field)
```

Comment:
This may lead to an error if it doesn`t exist

Reply:
then use this
if(array_key_first(current($attributes)) == $attr_field)

Reply:
@falcesch
It is not my responsibility to address the issues you're trying to introduce with this PR.

https://opensource.creativecommons.org/contributing-code/pr-guidelines/